flink-runtime/src/main/java/org/apache/flink/util/DualKeyLinkedMap.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.jobmaster.slotpool;
+package org.apache.flink.util;
```

Comment:
Not pretty sure why we move it to flink-core?

Reply:
it is in `flink-runtime` but true it should be in `package org.apache.flink.runtime.util;`